tests/languages/clojure/keyword_feature.test

```diff
@@ -0,0 +1,37 @@
+(def)
+(defmacro)
+(if)
+(do)
+(let)
+(var)
+(fn)
+
+----------------------------------------------------
+
+[
+	["punctuation", "("],
+  ["keyword", "def"],
```

Comment:
Could you indent those lines with a tab, please?

Reply:
Finished 😌